team/levels.tex

```diff
@@ -0,0 +1,105 @@
+\documentclass{article}
+\usepackage{graphicx}
+\usepackage{listings}
+\lstset{language=C}
+\graphicspath{ {images/} }
+\usepackage{hyperref}
+\hypersetup{
+    colorlinks=true,
+    linkcolor=blue,
+    filecolor=magenta,      
+    urlcolor=cyan,
+}
+\lstset{basicstyle=\sffamily}
+\usepackage{array}
+\usepackage[table]{xcolor}
+\usepackage{geometry}
+ \geometry{
+ a1paper,
+ left=20mm,
+ top=20mm,
+ }
+ 
+\begin{document} 
+\title{Engineering Levels}
+
+\section{Legend}
+ 
+The matrix below explains the requirements for engineering levels. Each level has to match at least one of the requirements for the level and the level before it.
+
+Not all skills go to level 6. This is determined by the focus of Teleport.
+For example, security skills are more relevant, as we are security oriented company, and we are interested in R\&D level 6 engineers, however OS/Tooling is not primary focus, so basic levels are satisfactory.
+As the level advances past 4, the focus shifts from technical skills to bringing products to market, as it's more relevant to Teleport as a for-profit startup.
+
+Skills are a mix of technical and communication skills, some references to security are based on the coding challenge.
```

Comment:
```suggestion
Skills are a mix of technical and communication abilities, some references to security are based on the coding challenge.
```
to avoid repetition.